draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -741,6 +819,10 @@
 
         <t/>
 
+	<!-- XXX JMH - in the PDU diagram below, the sub-type is one octet
+	instead of the two we've seen elsewhere?  Where are the various total
+	number fields defined?  Is this meant to be a new sub-sub-tlv? This
+	section isn't really complete.-->
         <figure anchor="Raw Load Sub-TLV"
                 title="Packets and Bytes Measurements Sub-TLV">
           <artwork><![CDATA[ 
```

Comment:
meant to be a sub-sub-TLV to be included in the VALUE field of the Raw-Measurement Sub-Type.

Reply:
Please answer whether this is a new sub-sub TLV. The text here isn't clear without context.

Reply:
The current text still makes this look like a sub-sub-tlv and the PDU format changes.